src/main/java/com/leverx/employeestat/rest/controller/AuthorizationController.java

```diff
@@ -38,9 +43,11 @@ public AuthorizationController(AuthorizationService authorizationService, Bindin
     public EmployeeDTO registerEmployee(@ApiParam(name = "Registration request", value = "Contains mandatory fields for registration of employee")
                                         @RequestBody
                                         @Valid RegistrationRequest request, BindingResult result) {
+        log.info("executing registerEmployee() method");
         if (result.hasErrors()) {
-            throw new NotValidRecordException("Fields of Employee have errors: " +
-                    bindingResultParser.getFieldErrMismatches(result));
+            NotValidRecordException e =  new NotValidRecordException("Fields of Employee have errors: " + bindingResultParser.getFieldErrMismatches(result));
+            log.error("Thrown exception", e);
```

Comment:
it has sense to log exceptions centrally in the exception handler instead of controller, to reduce non-functional code here